pom.xml

```diff
@@ -57,6 +57,12 @@
         <artifactId>jsonrpc4j</artifactId>
         <version>1.6.0</version>
       </dependency>
+
+      <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-core</artifactId>
+        <version>1.43.0</version>
```

Comment:
This version is not yet available in Maven Central. It will be released by end of this week. This PR will be merged after the release and when the build is successful.